minethd.cpp

```diff
@@ -33,7 +33,11 @@
 
 void thd_setaffinity(std::thread::native_handle_type h, uint64_t cpu_id)
 {
+#ifdef __MINGW64__
+	SetThreadAffinityMask(&h, 1ULL << cpu_id);
```

Comment:
You are not quite correct on theory side this is valid code in C or C++ (in fact it is quite useful for writing shellcodes)
```
#include <stdio.h>
int main()
{
	void* stack_top;
	stack_top = &stack_top;
	printf("%.16lX", (unsigned long)stack_top);
	return 0;
}
```
Do you mind if I put this one on hold until I can verify it myself?